manifests/init.pp

```diff
@@ -193,6 +193,12 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing. This requires Foreman 1.22+
 #
+# $foreman_puma_threads_min::     Minimum number of threads for puma
```

Comment:
Would it make sense to say this is only relevant when the Puma service is used but ignored when passenger is used?

Reply:
I also don't think we need the `foreman_` in front. These are already scoped by the puppet module itself.

Reply:
The related scoping is because there is a service and its named foreman so `foreman_service_` makes sense. If you want to tie this specifically to that, I would namespace them as `foreman_service_puma_threads_min` but I don't feel that its required to namespace them.

Reply:
> The related scoping is because there is a service and its named foreman so `foreman_service_` makes sense. If you want to tie this specifically to that, I would namespace them as `foreman_service_puma_threads_min` but I don't feel that its required to namespace them.
I am going to use `foreman_service_` just to match with the other foreman service params. It will be easy to debug/refer these params in future.